hera_stats/jackknives.py

```diff
@@ -306,15 +312,15 @@ def stripe_times(uvp, period=None, verbose=False):
         [uvp1, uvp2] = [uvp.select(times=t, inplace=False) for t in [t1, t2]]
 
         # Set metadata
-        uvp1.labels = "Period %.2f sec Even" % per
-        uvp2.labels = "Period %.2f sec Odd" % per
+        uvp1.labels = np.array(["Period %.2f sec Even" % per])
```

Comment:
It's not strictly necessary, but UVPSpec labels should normally be quite simple string identifiers, e.g. `period_1.00_even`. Think about them as dictionary keys, rather than as descriptions.